js/run_examples.js

```diff
@@ -0,0 +1,159 @@
+$(document).ready(function()
+{
+    if ($('body')[0].id == "Home")
+        return;
+
+    // only for std at the moment
+    if(!$('body').hasClass("std"))
+        return;
```

Comment:
Would be nice to avoid including the JS file at all on other pages, so there's no useless impact on load speed.